scripts/component-base.js

```diff
@@ -155,13 +176,14 @@ export default class ComponentBase extends HTMLElement {
    * use the data attr values as default for attributesValues
    */
   setInitialAttributesValues() {
-    const initialAttributesValues = { all: {} };
+    const initialAttributesValues = { all: { data: {} } };
 
-    this.Handler.observedAttributes.map((dataAttr) => {
-      const [, key] = dataAttr.split('data-');
-      const value = this.dataset[key];
+    this.dataAttributesKeys.forEach(({ noData, noDataCamelCase }) => {
+      const value = this.dataset[noDataCamelCase];
+      // TODO check this
```

Comment:
valid TODO? should we add a more elaborate description what to check here?

Reply:
Left over comment. Removed!